apps/server/swagger/dictionary-api.yml

```diff
@@ -19,10 +19,13 @@
                 type: string
                 description: A matching Dictionary Name defined on Dictionary Manager (Lectern)
                 required: true
-              version:
+              dictionaryVersion:
```

Comment:
to confirm for my own understanding: this is different from a schema version, right?

Reply:
it's different from schema version, changed variable name to `dictionaryVersion`